Added /storage endpoints to simulate localStorage and removed unused …

Sushmita-test opened the pull request:
…code

Sushmita-test committed:
Added /storage endpoints to simulate localStorage and removed unused code

LinusTyr reviewed (changes requested):
Great that you have added a class that acts as a request/response object.

You still need to remove the unused files though.

Sushmita-test committed:
Cleaned project and implemented storage endpoints

LinusTyr commented:
I don't seem to be getting through. 

Let me start over.

It is a good practice to remove unused code. Your recent changes has made some code (*.cs files) unused.
Some of these files are no longer used:
<img width="386" height="554" alt="image" src="https://github.com/user-attachments/assets/295386b7-1eb7-4df5-849f-99bba8e945a1" />
You should be able to figure out which ones and have them removed.

LinusTyr reviewed (changes requested):
See my comment https://github.com/SFX-IT-Orienteringskursen-VT25/assign3-addition-api/pull/31#issuecomment-3523773817

Sushmita-test committed:
Removed unused files and cleaned project

Sushmita-test committed:
Removed embedded git repository

Sushmita-test committed:
Removed .DS_Store

Sushmita-test commented:
@LinusTyr I removed all unused files and cleaned the project structure. The code compiles and works as expected. 
<img width="1429" height="784" alt="Screenshot 2025-11-21 at 10 52 40" src="https://github.com/user-attachments/assets/8ebf1a12-50bd-4075-b43a-7c9c967f958b" />

LinusTyr commented on Server/AdditionApi/Program.cs:
```diff
@@ -1,55 +1,46 @@
+app.Run();
+
+// Request/response model
+public record StorageItem(string Key, string Value);
```
I'd break this out into a separate file.

LinusTyr reviewed (approved):
Good job!
